src/assets/index.css

```diff
@@ -4,8 +4,13 @@
 }
 
 html, body, #background, #landing_container, #input_container, #result_container{
-  height:100vh;
-  width:100vw;
+  /* height:100vh; */
```

Comment:
uncomment this, 
I can guess it was hard to position your form, go through all the review
I received mal-aligned form, 
I would suggest you to add 100-150px top-padding to input_container, so that your design start below the logo
you may update margins of the form accordingly

![form_cont](https://user-images.githubusercontent.com/33171120/78473221-95ac1200-775c-11ea-99ab-886025126618.png)
(#landing_container and #result_container are hidden)